src/Parser/Warlock/Affliction/Modules/Features/LowMana.js

```diff
@@ -0,0 +1,56 @@
+import Analyzer from 'Parser/Core/Analyzer';
+import RESOURCE_TYPES from 'common/RESOURCE_TYPES';
+
+const MAX_MANA = 1100000;   // should be constant
+const LOW_MANA_THRESHOLD = 0.05;
```

Comment:
Minor thing, I think it might make more sense to set it to a specific value based on the cost of your spells instead of a percent. If your dots cost 33k per cast, going to 54k mana (<5%) isn't really different from 56k mana (>5%) since you'll only be able to cast 1 more spell anyway. If the threshold was 66k instead it would be exactly enough to cast 2 extra spells.

Reply:
I've talked to Motoko about this and basically the 5% threshold was chosen because all of our mana-based spells use at most 5% mana (Haunt), so the threshold is pretty much "to have enough mana for any *one* mana-based spell". I've decided to leave that as a percentage so it can be correct even if mana (and the cost) goes up.